modules/ROOT/nav.adoc

```diff
@@ -21,25 +21,25 @@
   *** xref:flow-management.adoc[Flow Management]
   *** xref:hybrid-manage-props.adoc[Managing Properties]
  ** xref:secure-application-properties.adoc[Safely Hiding Application Properties]
-* xref:managing-servers.adoc[Servers, Server Groups, and Clusters (Hybrid)]
+* xref:managing-servers.adoc[Servers, Server Groups, and Clusters]
```

Comment:
Since the hybrid icon remains in the topics, wondering why we are removing this bit of info from the titles. 
i.e. will this make searching from Google less productive?
I agree that the whole parens thing is awkward and it'll be good to get rid of the parens.

Reply:
Style Guide.